.github/workflows/invalidate-cache.yaml

```diff
@@ -0,0 +1,76 @@
+name: Invalidate cache on comment
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  actions: write
+  contents: read
+  issues: read
+  pull-requests: write
+
+jobs:
+  check-comment:
+    name: Invalidate cache
+    if: (github.event.issue.pull_request && startsWith(github.event.comment.body, 'Invalidate cache'))
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ github.token }}
+
+    steps:
+      - uses: xt0rted/pull-request-comment-branch@v2
+        id: comment-branch
+
+      - name: Set up repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+
+      - name: Check if user has write access
+        id: write_access
+        uses: prince-chrismc/check-actor-permissions-action@v3
+        with:
+          permission: write
+
+      - name: Get cache prefix
+        run: |
+          comment="${{ env.COMMENT }}"
+          cache_prefix="${comment#Invalidate cache }"
+          echo "CACHE_PREFIX=$cache_prefix" >> $GITHUB_ENV
+        env:
+          COMMENT: ${{ github.event.comment.body }}
+
+      - name: List caches
+        run: |
+          gh api repos/${{ github.repository }}/actions/caches | jq -c '{caches: [.actions_caches[] | select(.key | startswith("${{ env.CACHE_PREFIX }}"))]}' > caches.json
+
+      - name: Add comment
+        run: |
+          if [[ $(jq '.caches | length == 0' caches.json) == "true" ]]; then
+            echo 'No caches found matching prefix `${{ env.CACHE_PREFIX }}` 🤨' >> body.txt
+          else
+            CACHE_COUNT=$(jq '.caches | length' caches.json)
```

Comment:
First find CACHE_COUNT, then compare it to 0 in if? Now you're counting it twice

Reply:
True, didn't notice. Those `jq` calls are cheap anyway, so I'm inclined to leave it as is.